app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -197,6 +198,8 @@ export const FormAudience = ({
 
   const isDesktop =
     experiment.application === NimbusExperimentApplicationEnum.DESKTOP;
+  const isLiveRollout = experiment.isRollout && getStatus(experiment).live;
+  const isLocked = isLiveRollout && !getStatus(experiment).draft;
```

Comment:
Wait, we DO lock it if it's a rollout and it's live?  Shouldn't it be the other way around?  Also is the idea here that users can get back to this page, and see the fields being locked in some case where it's not a live rollout, ie a completed rollout or a live experiment?  I'm not sure that's the right approach?  Couldn't we set it up so that users can only navigate to the page in the case that it's a live rollout, and then they can get to the page altogether in the cases of

1) It's a draft of any type
2) It's a live rollout

and in both of those cases the fields are all unlocked, and in all other cases they just can't get to the page altogether?

Reply:
Oh I see this is meant as a no-op change and the additional logic to expose it for rollouts will come in a followup.  Sounds good!

Reply:
Yes, exactly :) The PRs following this will ensure that only the correct fields are unlocked for live rollouts